#goal definition
nav_msgs/Path path
float64 radius
---
#result definition
bool success
---
#feedback
int32 waypoint
geometry_msgs/Pose setPoint

